package.json

```
@@ -31,7 +31,8 @@
         "argparse": "^2.0.1",
         "assert": "^2.0.0",
         "axios": "^0.21.1",
-        "boa-sdk-ts": "^0.1.6",
+        "boa-sdk-ts": "^0.1.7",
```

```suggestion
        "boa-sdk-ts": "^0.1.8",
```

https://github.com/bosagora/boa-sdk-ts/pull/233
Please include this, too.